applications/luci-app-ddns/htdocs/luci-static/resources/view/ddns/overview.js

```diff
@@ -0,0 +1,993 @@
+'use strict';
+'require uci';
+'require rpc';
+'require fs';
+'require form';
+'require tools.widgets as widgets';
+
+var callGetLogServices, callInitAction, callReloadService,
+	callDDnsGetStatus;
+
+var time_res = {};
+time_res['seconds'] = 1;
+time_res['minutes'] = 60;
+time_res['hours'] = 3600;
+
+callGetLogServices = rpc.declare({
+	object: 'ddns',
+	method: 'get_services_log',
+	params: [ 'service_name' ],
+	expect: {  },
+});
+
+callReloadService = rpc.declare({
+	object: 'ddns',
+	method: 'reload_service',
+	params: [ 'service_name' ],
+	expect: {  },
+});
+
+callInitAction = rpc.declare({
+	object: 'luci',
+	method: 'setInitAction',
+	params: [ 'name', 'action' ],
+	expect: { result: false }
+});
+
+callDDnsGetStatus = rpc.declare({
+	object: 'ddns',
+	method: 'get_ddns_state',
+	expect: {  }
+});
+
+return L.view.extend({
+
+	callDDnsGetEnv: rpc.declare({
+		object: 'ddns',
+		method: 'get_env',
+		expect: {  }
+	}),
+	
+	callDDnsGetServicesStatus: rpc.declare({
+		object: 'ddns',
+		method: 'get_services_status',
+		expect: {  }
+	}),
+	
+	poll_status: function(map, data) {
+		var status = data[1], service = data[0], rows = map.querySelectorAll('.cbi-section-table-row[data-sid]'),
+			section_id, cfg_detail_ip, cfg_update, cfg_status, host, ip, last_update, 
+			next_update, service_status, reload, cfg_enabled, stop,
+			ddns_enabled = map.querySelector('[data-name="_enabled"]').querySelector('.cbi-value-field'),
+			ddns_toggle = map.querySelector('[data-name="_toggle"]').querySelector('button'),
+			action = status['_enabled'] ? 'stop' : 'start';
+
+		ddns_toggle.innerHTML = _(action.toUpperCase() + ' DDns')
```

Comment:
Constructs like these do not allow for proper translation, you always need to pass constant strings to the `_()` function.

`ddns_toggle.innerHTML = status['_enabled'] ? _('Stop DDNS') : _('Start DDNS')`